front/src/pages/back-office/products.jsx

```diff
@@ -0,0 +1,64 @@
+import BackOfficeBoard from "@/components/BackOfficeBoard"
+import BackOfficeNav from "@/components/BackOfficeNav"
+import Page from "@/components/Page"
+import { backoffice } from "@/constants"
+import { getProducts } from "@/utils/services/productServices"
+import { useEffect, useState } from "react"
+
+const ProductsBoard = () => {
+  const [selected, setSelected] = useState({})
+  const [isModalOpen, setIsModalOpen] = useState(false)
+  const [products, setProducts] = useState([])
+  const [itemId, setItemId] = useState(null)
+  const handleOpenModal = event => {
+    const id = event.currentTarget.getAttribute("data-id")
+    setItemId(id)
+    setIsModalOpen(!isModalOpen)
+  }
+
+  const handleChange = event => {
+    // More things to do when filter will be done.
+    const id = event.currentTarget.getAttribute("data-id")
+
+    if (id === "all") {
```

Comment:
This code can seem unreadable, can you refactor it ?

Reply:
I don't know how you want to me to refactor it its just a check for the id and if the "id" is not an id its "all" then I put all the entries in true and if "id" is an id only the one with an id is put to true